### PR TITLE
Dependabots 21 nov

### PR DESCRIPTION
**Pull request by alexjavabraz**
- dependabots alerts
#900 #901 #902 #903 #904 #905 #906 #907 #908 #909 #910 #912 #913 #914 #916 #917 #918 #920 #921 #922

### CODE REVIEW

**Review by leoiovlabs — DISMISSED** [review]
LGTM

File: Pipfile
Comment (lserra-iov) [inline Pipfile:1]: Why do we need this file?

**Review by annipi — APPROVED** [review]
Small suggestions about new `v3.27.5` of `github/codeql-action`

**Review by lserra-iov — DISMISSED** [review]

**Review by annipi — DISMISSED** [review]

**Review by leoiovlabs — APPROVED** [review]
LGTM

**Review by ronaldsg20 — APPROVED** [review]

**Review by lserra-iov — APPROVED** [review]

**Pull request merged**